flutter/example/lib/main.dart

```diff
@@ -146,6 +146,26 @@ class MainScaffold extends StatelessWidget {
               onPressed: () => Future.delayed(Duration(milliseconds: 100),
                   () => throw Exception('Throws in Future.delayed')),
             ),
+            RaisedButton(
+              child: const Text('Capture from FlutterError.onError'),
+              onPressed: () {
+                // modeled after a real exception
+                FlutterError.onError?.call(FlutterErrorDetails(
+                  exception: Exception('A really bad exception'),
+                  silent: false,
+                  context: DiagnosticsNode.message('while handling a gesture'),
+                  library: 'gesture',
+                  informationCollector: () => [
+                    DiagnosticsNode.message(
+                        'Handler: "onTap" Recognizer: TapGestureRecognizer'),
+                    DiagnosticsNode.message(
+                        'Handler: "onTap" Recognizer: TapGestureRecognizer'),
+                    DiagnosticsNode.message(
+                        'Handler: "onTap" Recognizer: TapGestureRecognizer'),
+                  ],
+                ));
+              },
+            ),
```

Comment:
the example `Flutter error : Scaffold.of()` already triggers a `FlutterError.onError` error, isnt that enough? is it common that people trigger `FlutterError.onError` manually too?

Reply:
> the example Flutter error : Scaffold.of() already triggers a FlutterError.onError error, isnt that enough?

This is just to test how it looks in the Sentry UI when it has a long trail of information, which `Flutter error : Scaffold.of()` doesn't have (as seen in the screenshot of the PR description). We can remove it before merging.

> is it common that people trigger FlutterError.onError manually too?

No, but it's not uncommon in libraries.